### DISCUSSION TITLE
Welcome to global-copilot-summit Discussions!

### DISCUSSION THREAD

**Discussion by kamalshree** [microsoft/global-copilot-summit#1] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to answer all questions related to Copilot. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Any Issues that you need help with.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do .

**Comment by protonerdy** [comment]
Hey, I've filled the form but I haven't received any confirmation or any of that sort on my email. When can I expect to get it?

**Comment by amit-jesani** [comment]
I registered myself for Mumbai event but not received any confirmation yet.

**Comment by saisuseelmohan2003** [comment]
Mam, i registered for event in Hyderabad. But I didn't get any confirmation mail

**Reply by shashankch2003** [reply]
we are not receiving confirmation mail mam

**Comment by shashankch2003** [comment]
could you help me i got registered for global esummit Hyderabad but didn't receive the confirmation mail yet

**Comment by protonerdy** [comment]
Hey everyone, I attended the event organised at Gurgaon on 27th. I received my confirmation around ~24 hours prior to the event. Maybe you'll receive it then too.

**Comment by shashankch2003** [comment]
I have not received the confirmation email yet. As the team lead, I'm wondering why my team members received theirs.
